trilium-note/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  server:
```

Comment:
Is there a reason you've not used/included the app proxy here?

Reply:
Yes. The main reason the app was breaking was the app_proxy tag. As the is being used as a server it is now running fine. Been stuck on it for weeks then saw the Plex app is facing the same issue on Umbrel and I took the reference from them.

Reply:
Can you elaborate a little more on what was broken?

Reply:
YES. I can share screenshots of the error.

![image](https://user-images.githubusercontent.com/43396684/196074493-0af9e9c4-a9dd-4b13-8646-ccfdccab6200.png)

this was the compose file
```
version: "3.7"

services:
  app_proxy:
    environment:
      APP_HOST: $APP_TRILIUM_NOTE_IP
      APP_PORT: 8080

  server:
    image: zadam/trilium:latest
    restart: on-failure
    user: "1000:1000"
    volumes:
      - ${APP_DATA_DIR}/home/node/trilium-data
    networks:
      default:
        ipv4_address: $APP_TRILIUM_NOTE_IP
        ```

Reply:
Plex has the same issue I have read while going through their repo. Maybe it's the same issue here

Reply:
The error you've shared looks like an issue related to yaml formatting. I've just pushed a commit to add the app proxy and it appears to be working fine.